.github/workflows/workflow.yml

```diff
@@ -7,6 +7,9 @@ on:
       - main
   pull_request:
 
+env:
+  IS_CI_WORKFLOW: True
+
```

Comment:
this is used to skip tests on CI - used to skip public dht tests (libp2p) right now, which you can run locally just straightforwardly:
```
pytest -s packages/valory/connections/test_libp2p/tests/test_dht.py
...
==================================================================================== 16 passed, 1 warning in 346.34s (0:05:46) ====================================================================================
```

Reply:
The tests passed on my local as well.

Reply:
and FYI: I intend to use this same `IS_CI_WORKFLOW` env var for the test that I've been (and still am) working on to test the robustness of our ACN